platform/feature-api/communication/postoffice/src/protocol.ts

```diff
@@ -2,19 +2,19 @@
  * This module defines the protocol used for translating function calls into bare objects. Users rarely need to use this
  * module directly.
  *
- * The [[endpointClient]] and [[endpointServer]] functions speak a simple protocol:
+ * The [[backendClient]] and [[backendServer]] functions speak a simple protocol:
  *
- * - each call is represented as an [[EndpointRequestPart]]
- * - a chain of calls is represented as an [[EndpointRequest]]
- * - a response is any kind of value ([[EndpointResponse]])
+ * - each call is represented as an [[BackendRequestPart]]
+ * - a chain of calls is represented as an [[BackendRequest]]
+ * - a response is any kind of value ([[BackendResponse]])
```

Comment:
Again, terminology: a "call" is a "part" of a "request" (when call and request are essentially synonyms). In this case this was in the original code, but still, as long as we're changing it...

Reply:
I'd say we best leave it as it was, given how we don't fully understand how they used terminology there and where else they use terms referred to here, like "call". Could be investigated now, but makes the whole thing a bit bigger if we try to change more terminology than just `s/endpoint/backend`.